### PR TITLE
Problem Suggestion: Add "Pairwise Modulo" to Gold - Point Update Range Sum

### PR DESCRIPTION
**Pull request by jeffkmeng**
User `R1WbsBWeBlaamf6nKdTxydN3aAH3` suggested adding the problem [Pairwise Modulo](https://codeforces.com/contest/1553/problem/F) to the `practice` table of the module [Gold - Point Update Range Sum](https://usaco.guide/gold/PURS#problemlist-practice).

**Automatically Generated JSON:**
```json
{
  "uniqueId": "cf-1553F",
  "name": "Pairwise Modulo",
  "url": "https://codeforces.com/contest/1553/problem/F",
  "source": "CF",
  "difficulty": "Very Hard",
  "isStarred": false,
  "tags": [
    "Math",
    "Harmonic Series",
    "PURS"
  ],
  "solutionMetadata": {
    "kind": "autogen-label-from-site",
    "site": "CF"
  }
}
```
**Additional Notes**: None

*This PR was automatically generated from a user-submitted problem suggestion on the USACO guide.*

### CODE REVIEW

**Comment by jessechoe10** [comment]
This problem rec pr sorta conflicts with [this](https://github.com/cpinitiative/usaco-guide/pull/1979) pr... Which one should we choose?

**Comment by timothygao8710** [comment]
I'm p sure the sqrt decomp sol to this problem is not intended, and can give tle without careful optimization.

File: content/4_Gold/PURS.problems.json [hunk 1]
```diff
@@ -115,6 +115,19 @@
...
       }
+    },
+    {
+      "uniqueId": "cf-1553F",
+      "name": "Pairwise Modulo",
+      "url": "https://codeforces.com/contest/1553/problem/F",
+      "source": "CF",
+      "difficulty": "Very Hard",
+      "isStarred": false,
+      "tags": ["Math", "Harmonic Series", "PURS"],
```
Comment (dongliu0426) [inline content/4_Gold/PURS.problems.json:126]:
```suggestion
      "tags": ["NT", "PURS"],
```

**Comment by dongliu0426** [comment]
I think this problem should be "Normal". @jessechoe10 @timothygao8710 opinions?

**Comment by jessechoe10** [comment]
> I think this problem should be "Normal". @jessechoe10 @timothygao8710 opinions?

Is `2300` normal for gold? I think it should be "Hard" idk.

**Comment by timothygao8710** [comment]
I mean I feel that it should be more or less evaluated relative to the rest of gold problems, and imo it's definitely harder than https://cses.fi/problemset/task/1734(normal), for example. Although I do agree with changing it to ```hard```.

**Pull request closed without merging**